lib/support/cli.js

```diff
@@ -622,11 +637,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
   }
 
   if (argv.android) {
-    set(
-      argv,
-      'chrome.android.package',
-      get(argv, 'chrome.android.package', 'com.android.chrome')
-    );
+    if (argv.browser != 'firefox') {
+      // Default to Chrome Android.
+      set(
+        argv,
+        'chrome.android.package',
+        get(argv, 'chrome.android.package', 'com.android.chrome')
+      );
+    } else {
+      // Default to GeckoView example.
+      set(
+        argv,
+        'firefox.android.package',
+        get(argv, 'firefox.android.package', 'org.mozilla.geckoview_example')
```

Comment:
I'm curious why are we defaulting to geckoview_example instead of Fenix?  Is this mainly just for mach usage?

Reply:
Yes.  We're totally between horses: `org.mozilla.firefox` is authoritative for everybody _not building Gecko_, `org.mozilla.fenix` is current but short-lived, and `org.mozilla.geckoview_example` makes the most sense for `mach` commands.  I think we can change this pretty easily 'cuz all consumers should set it explicitly.